spec/core/chain/schemas/FinalizedBlockDTO.yml

```diff
@@ -0,0 +1,16 @@
+type: object
```

Comment:
Is this the same schema than the finalized block WS payload? We should move this file to the top level schema folder